tests/test_main.py

```diff
@@ -141,6 +141,53 @@ def test_lint_no_multiprocessing(fake_repository, stringio, multiprocessing):
     assert ":-)" in stringio.getvalue()
 
 
+def test_list(fake_repository, stringio):
+    """List all files and their copyright information."""
+    os.chdir(str(fake_repository))
+    result = main(["list"], out=stringio)
+
+    lines = [line.rstrip() for line in stringio.getvalue().split("\n")]
+    expected_lines = [
+        "File                    License",
```

Comment:
Maybe there should be an option to skip the headers.
Makes things easier e.g. if one processes the results by a script.